script.js

```diff
@@ -0,0 +1,174 @@
+/*https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_grid_layout/Basic_concepts_of_grid_layout*/
+
+let newGameButton = document.getElementById('newGameButton');
+newGameButton.style.display = 'none';
+
+let winLoseDiv = document.getElementById('winLoseDiv');
+winLoseDiv.style.display = 'none';
+
+
+/*Choose Word*/
+document.getElementById('submitPhraseButton').addEventListener('click', stringToArrayHandler);
+let letterArray = [];
+
+let imageLocation = document.getElementById('img');
+
+let currentImageNumber = 1;
+
+let regex = /[a-zA-Z]/g;
+let slotsToFill = 0;
+
+let filledSlots = 0;
+
+
+let letterBlockArray = ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z'];
+
+
+
+function stringToArrayHandler(event) {
+//letters become clickable
+  addLetterBlockEventListeners();
+
+  let string = document.getElementById('phrase').value.toUpperCase();
+  slotsToFill = string.match(regex).length;
+  console.log("slotsToFill = " + slotsToFill);
+  letterArray = string.split("");
+  console.log(letterArray);
+  console.log(letterArray[3]);
+  // make text box and button hidden once 'Submit Phrase' button has been clicked
+  document.getElementById('phrase').style.display = 'none';
+  document.getElementById('submitPhraseButton').style.display = 'none';
+
+  ArrayToDivCreation(letterArray);
+}
+const currentDiv = document.getElementById("letters");
```

Comment:
this global variable is oddly placed.
pinched between 2 functions it's easy to miss.
maybe it belongs with the other global variables you initialized up top.